pubspec.lock

```diff
@@ -436,10 +452,10 @@ packages:
     dependency: "direct main"
     description:
       name: hooks_riverpod
-      sha256: "45b2030a18bcd6dbd680c2c91bc3b33e3fe7c323e3acb5ecec93a613e2fbaa8a"
+      sha256: "169d7bf1d61c749c065e09388841906b2ff963794b932867372ed4f423a8d877"
       url: "https://pub.dev"
     source: hosted
-    version: "2.5.1"
+    version: "3.0.0-dev.3"
```

Comment:
sometime we don't want to use dev/alpha versions in prod -whats' the purpose of this change? riverpod 3.o probably will be exciting but shoud probabl be its own PR

Reply:
This is the reason:
Bug: https://github.com/rrousselGit/riverpod/issues/3009
Fix: https://github.com/rrousselGit/riverpod/pull/3030

I couldn't write really tests for these providers without upgrading to riverpod 3.0.0